src/main.cpp

```diff
@@ -4581,9 +4605,8 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
                 }
 
 
-                if (!pushed) {
+                if (!pushed)
```

Comment:
I thought the style guide for BTC was to have curly braces after the line break?